src/collectors/mongodb/mongodb.py

```diff
@@ -147,16 +152,15 @@ def collect(self):
                     self.config['ssl'] = str_to_bool(self.config['ssl'])
 
                 if ReadPreference is None:
-                    conn = pymongo.Connection(
+                    conn = pymongo.MongoClient(
```

Comment:
This change allowed us to use pymongo 3.x, which [changed the client class constructor when moving from 2.x to 3.0](http://api.mongodb.org/python/current/changelog.html#mongoclient-changes). There should be no need to upgrade the pymongo client if they are using a relatively up to date version of 2.x (Connection has been depreciated since 2.5 at least). However, users should note that the network_timeout value is now in milliseconds when configuring this collector.